<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-GODS5UC" name="Default test" tests="4" failures="1" timestamp="17 Jun 2020 10:17:08 GMT" time="68.592" errors="0">
  <testcase name="Loadsite" time="14.311" classname="com.yatra.testCase.YatraSiteTestCase"/>
  <testcase name="login" time="10.421" classname="com.yatra.testCase.YatraSiteTestCase"/>
  <testcase name="searchCab" time="15.994" classname="com.yatra.testCase.YatraSiteTestCase"/>
  <testcase name="ExcelWrite" time="0.741" classname="com.yatra.testCase.YatraSiteTestCase">
    <failure type="java.lang.StringIndexOutOfBoundsException" message="String index out of range: -1">
      <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(Unknown Source)
	at com.yatra.utilities.Excel.findLowPrice(Excel.java:104)
	at com.yatra.utilities.Excel.writeCabInfo(Excel.java:34)
	at com.yatra.webPages.CabList.writeData(CabList.java:37)
	at com.yatra.testCase.YatraSiteTestCase.ExcelWrite(YatraSiteTestCase.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
    </failure>
  </testcase> <!-- ExcelWrite -->
</testsuite> <!-- Default test -->
